cypress/integration/treeFromObject.spec.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+Cypress.Commands.add('haveChildren', (value) => {
```

Comment:
since number 2 is hardcoded, how about we explain that in the name?

hasTwoChildren?

Reply:
has sense) done